src/nbl/asset/interchange/CIESProfileLoader.h

```diff
@@ -0,0 +1,160 @@
+// Copyright (C) 2018-2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef __NBL_ASSET_C_IES_PROFILE_LOADER_H_INCLUDED__
+#define __NBL_ASSET_C_IES_PROFILE_LOADER_H_INCLUDED__
+
+#ifdef _NBL_COMPILE_WITH_IES_LOADER_
+
+#include "nabla.h"
+#include "nbl/asset/ICPUImage.h"
+#include "nbl/asset/ICPUShader.h"
+#include "nbl/video/IGPUShader.h"
```

Comment:
`nabla.h` is for external projects, and `asset` namespace is not allowed to include `video`